packages/eui/src/components/avatar/avatar.tsx

```diff
@@ -9,6 +9,8 @@
 import React, { HTMLAttributes, FunctionComponent, useMemo } from 'react';
 import { CommonProps, ExclusiveUnion } from '../common';
 import classNames from 'classnames';
+import { warnIfContrastBelowMin, wcagContrastMin } from '../../services/color/contrast';
+
```

Comment:
I see it's like this:

```suggestion
import {
  warnIfContrastBelowMin,
  wcagContrastMin,
} from '../../services/color/contrast';
```